docs/docs/en/guides/features/qa.md

```diff
@@ -0,0 +1,105 @@
+---
+{
+  "title": "QA",
+  "titleTemplate": ":title · Features · Guides · Tuist",
+  "description": "AI-powered testing agent that tests your iOS apps automatically with comprehensive QA coverage."
+}
+---
+# QA {#qa}
+
+> [!IMPORTANT] EARLY PREVIEW
+> Tuist QA is currently in early preview. Sign up at [tuist.dev/qa](https://tuist.dev/qa) to get access.
+
+Building high-quality mobile apps requires extensive testing, but manual QA testing is time-consuming, expensive, and difficult to scale. Tuist's QA agent autonomously navigates through your app, identifies UI elements, performs realistic user interactions, and reports issues it discovers – helping you catch bugs and usability issues early in your development cycle.
+
+## Prerequisities {#prerequisites}
+
+To start using Tuist QA, you need to:
```

Comment:
Why are those needed? Without context, it can be confusing. I'd say something along the lines:

Tuist QA is an agent that runs testing session and for that we need a "trigger" and the "object" to test, for that, you'll need to:

- Upload previews (.app) from your PR commits that the agent can use for testing. 
- Integrate the account with GitHub so can trigger the sessions as a result of actions there.